hazelcast/src/main/java/com/hazelcast/internal/jmx/suppliers/LocalMultiMapStatsSupplier.java

```diff
@@ -33,7 +33,7 @@ public LocalMultiMapStatsSupplier(MultiMap multiMap) {
 
     @Override
     public LocalMultiMapStats getEmpty() {
-        return new LocalMultiMapStatsImpl();
+        return new LocalMultiMapStatsImpl(false);
```

Comment:
What is the purpose of this constructor parameter?

Reply:
I "cache" of statistics are enabled (from config) for statistic objects as I ask the flag each rendering cycle.